app/src/main/java/nl/hogeschoolrotterdam/projectb/MemoryEditActivity.java

```diff
@@ -107,7 +108,8 @@ protected void onCreate(Bundle savedInstanceState) {
                     new Date(calendar.getTimeInMillis()),
                     "",
                     "",
-                    media
+                    media,
+                    R.drawable.ic_map_adefaultl
```

Comment:
There seems to be a typo here. It should be `ic_map_adefault` and not `ic_map_adefaultl`

Reply:
done